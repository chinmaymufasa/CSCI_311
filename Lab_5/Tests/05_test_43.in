2
4
5
7
2
123
5
7
2
16
5
7
2
72
5
7
2
115
5
7
2
69
5
7
2
28
5
7
2
155
5
7
2
145
5
7
2
17
5
7
2
98
5
7
2
57
5
7
2
96
5
7
2
161
5
7
2
16
5
7
2
105
5
7
2
47
5
7
2
84
5
7
2
155
5
7
2
192
5
7
-1
