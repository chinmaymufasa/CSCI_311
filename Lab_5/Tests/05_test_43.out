4 
4
4 123 
4
4 16 123 
4
4 16 72 123 
4
4 16 72 115 123 
4
4 16 69 72 115 123 
4
4 16 28 69 72 115 123 
4
4 16 28 69 72 115 123 155 
4
4 16 28 69 72 115 123 145 155 
4
4 16 17 28 69 72 115 123 145 155 
4
4 16 17 28 69 72 98 115 123 145 155 
4
4 16 17 28 57 69 72 98 115 123 145 155 
4
4 16 17 28 57 69 72 96 98 115 123 145 155 
4
4 16 17 28 57 69 72 96 98 115 123 145 155 161 
4
4 16 16 17 28 57 69 72 96 98 115 123 145 155 161 
4
4 16 16 17 28 57 69 72 96 98 105 115 123 145 155 161 
4
4 16 16 17 28 47 57 69 72 96 98 105 115 123 145 155 161 
4
4 16 16 17 28 47 57 69 72 84 96 98 105 115 123 145 155 161 
4
4 16 16 17 28 47 57 69 72 84 96 98 105 115 123 145 155 155 161 
4
4 16 16 17 28 47 57 69 72 84 96 98 105 115 123 145 155 155 161 192 
4
